Problem_1.java

```diff
@@ -0,0 +1,20 @@
+// Time Complexity : O(n) n is the rows of houses
+// Space Complexity:)O(1)
+// Did this code successfully run on Leetcode :yes
+// Any problem you faced while coding this : yes
+// Your code here along with comments explaining your approach
+//DP
```

Comment:
mention the problems that were faced during solving this problem. Can help accordingly.